docker-compose.yml

```diff
@@ -12,37 +12,37 @@ services:
     volumes:
       - ./auth/keycloak/config/realm-export.json:/tmp/realm-export.json
     ports:
-      - 8080:8080
+      - ${KEYCLOAK_PORT:-8080}:8080
     networks:
       - pims
 
   ####################### Database #######################
   database:
     restart: on-failure:1
-    container_name: api-db
+    container_name: pims-database
```

Comment:
I'm always suspicious of doing renames like this, because if an old reference is left somewhere it could break something important.

Have you tried this with a fresh install?

Reply:
I have done a fresh install.